lib/tasks/analytics.rake

```diff
@@ -26,4 +26,16 @@ namespace :analytics do
       end
     end
   end
+
+  desc "Delete old export files (specify the number to keep with EXPORT_FILE_LIMIT)"
+  task :delete_old_files do
+    path = ENV['EXPORT_PATH'] || 'data'
```

Comment:
This is so that you don't need to specify a path when running locally.. it will just put the generated file in you `<app>/data` folder and then in prod we specify the path in the jenkins job.. :)